RELEASE/scripts/autoscend/auto_quest_level_11.ash

```diff
@@ -1522,16 +1522,13 @@ boolean L11_mauriceSpookyraven()
 
 boolean L11_redZeppelin()
 {
-	if (internalQuestStatus("questL11Shen") < 8)
-	{
-		return false;
-	}
-
+	//if quest not started yet, or if you already cleared the protesters.
```

Comment:
Also no. Revert this too. Go read the wiki or the Copperhead code for why you might want to do the Shen part of the quest first.

Reply:
commit ac52148

Reply:
This is still unnecessarily complicated. You can achieve this the same way it is achieved already in the existing code simply by changing this condition to `my_level() != get_property("auto_powerLevelLastLevel").to_int()`. See `LX_handleSpookyravenFirstFloor()` for multiple uses of this.

Reply:
change made

Reply:
Wouldn't `my_level() != get_property("auto_powerLevelLastLevel").to_int()` waste 1 adv on power leveling before bypassing this? more if you level up mid doing the quest?

Reply:
You raise a good point. At the end of `doTasks()` we do the following
```
	if(my_level() != get_property("auto_powerLevelLastLevel").to_int())
	{
		set_property("auto_powerLevelLastLevel", my_level());
		return true;
	}
```
which I always assumed meant would allow `LX_attemptPowerLevel()` to run on the next loop iteration. But looking at that function it doesn't seem to check that...so I have no idea how or why this works.
There's a lot of spaghetti code still in here from when it was cc_ascend. I'll have to check a session log to see how this stuff all hangs together. It certainly seems to work but it's not obvious to me right now.

Reply:
Fair enough... actually you know what. I am fine with just using `my_level() != get_property("auto_powerLevelLastLevel").to_int()` for now, accepting the **possibility** of losing an adventure or two. and having a dedicated PR for just that in the future if we decide to improve it.

Reply:
And in that case. is there anything else that needs changing with this PR? if not I'd like to merge it.

Reply:
OK I investigated this and there's a lot to fix (oh god so much spaghetti code) but I agree with you, losing an adventure or two here and there isn't going to kill anyone for now.

I'll clean all this stuff up in a PR once I'm done with the level 8 quest stuff (and some minor bugfixes in various other places I have pending).

Reply:
> And in that case. is there anything else that needs changing with this PR? if not I'd like to merge it.

Fine by me if @jeparo requested changes have been addressed.